core/iwasm/interpreter/wasm_loader.c

```diff
@@ -7436,7 +7825,222 @@ wasm_loader_prepare_bytecode(WASMModule *module, WASMFunction *func,
 #endif
                 break;
             }
+#if WASM_ENABLE_EXCE_HANDLING != 0
+            case WASM_OP_THROW:
+            {
+                SET_CUR_BLOCK_STACK_POLYMORPHIC_STATE(true);
+
+                BranchBlock *cur_block = loader_ctx->frame_csp - 1;
+
+                uint8 label_type __attribute__((unused)) =
```

Comment:
Not sure whether `__attribute__((unused))` will cause compiler warning/error, e.g., in windows platform. How about remove it and use `(void)label_type`?

Reply:
Updated code according to recommendation